src/Elastic.Apm/Elastic.Apm.csproj

```diff
@@ -11,15 +11,11 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="System.Diagnostics.PerformanceCounter" Version="4.5.0" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.0.0" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.4.0" />  
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.6.0" />
```

Comment:
This is TBD - we recently changed this and increased to `4.4.0` only for `net461`. We should double-check if using `4.6.0` does not cause trouble to apps using the agent.

The newer version is needed, because `Activity.IdFormat` does not exist in older versions.